.dockerignore

```diff
@@ -0,0 +1,6 @@
+node_modules
```

Comment:
I wonder if this should just be a whitelist

```
*
!src/
!test/
[etc]
```

Reply:
Why'd you think that? There are a few well known build artifacts `node_modules` in particular that can cause issues when copied into the container so explicitly ignoring them seems more favourable to ignoring everything then only adding currently known good things in. The known good things seem like they would change more often than the known bad things.

Reply:
I have the opposite view - it seems like the bad will change more than the good.  I guess I also prefer things being explicit - if I'm interpreting correctly, the whitelist would explicitly say what was relevant for the docker build.  For example, you wouldn't accidentally copy docs unnecessarily unless you explicitly added a line for it.  I don't have a strong opinion here though

Reply:
That's a fair point, and the less in the container image the better. OK, you've persuaded me, I'll switch to the whitelist ;)